courses/intro-to-spark/book/01_Overview.rst

```diff
@@ -228,48 +232,42 @@ attempts to update the global variable ``Call_Count``.
           return X + 1;
        end Incr_And_Log;
 
-    end Test_Spark_Side_Effect;
+    end Side_Effects;
 
 
 No aliasing of names
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Another restriction imposed in the SPARK subset concerns aliasing. We say
-that two names are aliased if they refer to the same object. Since access
-types are not allowed in SPARK, aliasing can only occur as part of the
-parameter passing in a procedure call. As a consequence, when a procedure
-is called, SPARK makes sure that no :ada:`out` or :ada:`in out` parameter
-is aliased with either an other parameter of the procedure or a global
-variable updated in the procedure's body.
+Another restriction imposed in the SPARK subset concerns aliasing. We say that
+two names are aliased if they refer to the same object. Since access types
+(pointers in Ada) are not allowed in SPARK, aliasing can only occur as part of
```

Comment:
In general, we're avoiding terminologies from other languages. If you fear, however, that the concept of access types is not clear here, maybe a link to the _Intro to Ada_ course would be nice.

Reply:
since we're also targeting people who don't know Ada, I'd rather be clear about what access types are. I'll add a link to the intro-to-ada section.

Reply:
@gusthoff I think "pointer" is an acceptable terminology. It is much more general than C/C++, in fact Ada is pretty much the odd duck here.

@yannickmoy a link to https://en.m.wikipedia.org/wiki/Pointer_(computer_programming) on this link would still be an appreciated addition there I think.